code/game/objects/structures/watercloset.dm

```diff
@@ -322,8 +331,8 @@
 	O.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
 	if(isitem(O))
 		var/obj/item/I = O
-		I.acid_level = 0
-		I.extinguish()
+
+
```

Comment:
If you're going to remove this, then also remove the first two lines. They don't do anything if the latter two lines are gone.